src/user/user.spec.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env jest */
+
+import { omit } from "lodash";
+import { testConnection, xo } from "../util";
+
+const simpleUser = {
+  email: "wayne3@vates.fr",
+  password: "batman",
+};
+
+describe("user", () => {
+  afterAll(async () => {
+    await xo.deleteAllUsers();
```

Comment:
The idea behind these helpers is to not forget calling it, maybe it should be automatically defined in an `afterEach`?